Sources/VersionCheck/Helpers/RemoteConfigHelper.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  File.swift
+//  
+//
+//  Created by Emad Ghorbania on 11/11/2022.
+//
+
+import Foundation
+import FirebaseRemoteConfig
```

Comment:
I Recently did a RemoteConfig implementation on toldkontrol for feature flags. This seems to be for the specific versioning for updates. But later on I guess we can add that remoteConfig featureFlag implementation I have to our shared library perhaps. They don't need to be mixed together Is just what I'm thinking. What do you think Emad?

Reply:
Let's talk about it tomorrow, but it seems it's too specific to be on the shared library.